app/src/main/kotlin/com/exallium/todoapp/MainActivity.kt

```diff
@@ -59,4 +66,12 @@ class MainActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
```

Comment:
got it. changed. for some reason Android Studio overrode the function with a null initially. i wonder why?